nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -304,6 +319,9 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
     }
     handler.onMatchMethodInvocation(this, tree, state, methodSymbol);
     // assuming this list does not include the receiver
+    if (config.getAutoFixConfig().MAKE_CALL_GRAPH_ENABLED) {
```

Comment:
As mentioned before: all these should be guarded based on whether autofix mode is enabled in the first place or not